content/en/docs/concepts/security/security-checklist.md

```diff
@@ -167,9 +167,7 @@ to only allow a restricted set of syscalls since 2016 from
 [Docker Engine 1.10](https://www.docker.com/blog/docker-engine-1-10-security/),
 but Kubernetes is still not confining workloads by default. The default seccomp
 profile can be found [in containerd](https://github.com/containerd/containerd/blob/main/contrib/seccomp/seccomp_default.go)
-as well. Fortunately, [Seccomp Default](/blog/2021/08/25/seccomp-default/), a
-new alpha feature to use a default seccomp profile for all workloads can now be
-enabled and tested.
+as well. Fortunately, [Seccomp Default](/docs/tutorials/security/seccomp/) is now generally available (GA) in Kubernetes 1.27, and it can be enabled and tested to use a default seccomp profile for all workloads.
```

Comment:
What do you think of this?
```suggestion
but Kubernetes {{< skew currentVersion >}} defaults to not confining workloads with seccomp.
Similarly, the containerd container runtime has a [mechanism](https://github.com/containerd/containerd/blob/54732fa9fb921b98f4fc5d96025307cc807f04d9/contrib/seccomp/seccomp_default.go)
for seccomp defaulting.

In Kubernetes, you can use [Seccomp profile defaulting](/docs/tutorials/security/seccomp/#enable-the-use-of-runtimedefault-as-the-default-seccomp-profile-for-all-workloads)
to define a seccomp profile to use if a workload doesn't specify one.
However, you need to turn that on. The mechanism to set a default seccomp profile
is only used if you enable it.
```

Reply:
483995a1b536287c290e435b2297ad352b332bae

Reply:
The mention of “Kubernetes 1.27” will end up looking dated. We prefer [timeless](https://developers.google.com/style/timeless-documentation) documentation.